bug: pages scaffold ws_urls.rs.tpl does not compile — fn returns () but macro expects router

## Summary

`reinhardt-admin startapp <name> --with-pages` on rc.18 generates a `ws_urls.rs` whose body does not compile because `#[url_patterns(mode = ws)]` wraps the body with `.with_namespace(...)`, but the scaffold body returns `()`.

## Reproduction

```bash
reinhardt-admin startproject blog --with-pages
cd blog
reinhardt-admin startapp posts --with-pages
# Inspect generated src/apps/posts/ws_urls.rs — verbatim template:
```

```rust
use reinhardt::url_patterns;
use crate::config::apps::InstalledApp;

#[url_patterns(InstalledApp::posts, mode = ws)]
pub fn ws_url_patterns() {
    // Register WebSocket consumers here.
    // Example: .consumer(\"chat/\", chat_consumer)
}
```

```
cargo check
```

## Expected

`cargo check` succeeds on a freshly scaffolded pages project.

## Actual

```
error[E0599]: no method named \`with_namespace\` found for unit type \`()\` in the current scope
  --> src/apps/posts/ws_urls.rs:11:1
   |
11 | #[url_patterns(InstalledApp::posts, mode = ws)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   = note: this error originates in the attribute macro \`url_patterns\`
```

## Root cause

In `crates/reinhardt-core/macros/src/url_patterns.rs`, `build_wrapper_and_assertion` emits:

\`\`\`rust
let __router = (|| #fn_block)();
__router.with_namespace(...)
\`\`\`

The `unified` and `server`/`client` templates return \`UnifiedRouter::new()\` (or similar) so \`with_namespace\` resolves. But `ws_urls.rs.tpl` leaves the body empty, so \`__router\` is \`()\` and \`with_namespace\` is unresolved.

The ws unit test in the same file uses the correct shape:

```rust
pub fn ws_url_patterns() -> WebSocketRouter {
    WebSocketRouter::new().consumer(handler_a).consumer(handler_b)
}
```

## Proposed fix

Change `crates/reinhardt-commands/templates/app_pages_template/ws_urls.rs.tpl` (and the workspace variant) to:

```rust
use reinhardt::url_patterns;
use reinhardt::WebSocketRouter;
use crate::config::apps::InstalledApp;

#[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
pub fn ws_url_patterns() -> WebSocketRouter {
    WebSocketRouter::new()
    // Register WebSocket consumers here.
    // Example: .consumer(chat_consumer)
}
```

## Environment

- `reinhardt-web` version: `0.1.0-rc.18`
- Rust toolchain: stable (edition 2024)

--- a/crates/reinhardt-commands/templates/app_pages_template/ws_urls.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/ws_urls.rs.tpl
@@ -10,20 +10,24 @@
 //!
 //! ```rust,ignore
 //! use reinhardt::url_patterns;
+//! use reinhardt::WebSocketRouter;
 //! use crate::config::apps::InstalledApp;
 //!
 //! #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
-//! pub fn ws_url_patterns() {
-//!     // Register consumers via .consumer("path/", handler)
+//! pub fn ws_url_patterns() -> WebSocketRouter {
+//!     WebSocketRouter::new()
+//!     // Register consumers via .consumer(handler)
 //! }
 //! ```
 
 use reinhardt::url_patterns;
+use reinhardt::WebSocketRouter;
 
 use crate::config::apps::InstalledApp;
 
 #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
-pub fn ws_url_patterns() {
+pub fn ws_url_patterns() -> WebSocketRouter {
+    WebSocketRouter::new()
     // Register WebSocket consumers here.
-    // Example: .consumer("chat/", chat_consumer)
+    // Example: .consumer(chat_consumer)
 }

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/ws_urls.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/ws_urls.rs.tpl
@@ -10,20 +10,24 @@
 //!
 //! ```rust,ignore
 //! use reinhardt::url_patterns;
+//! use reinhardt::WebSocketRouter;
 //! use crate::config::apps::InstalledApp;
 //!
 //! #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
-//! pub fn ws_url_patterns() {
-//!     // Register consumers via .consumer("path/", handler)
+//! pub fn ws_url_patterns() -> WebSocketRouter {
+//!     WebSocketRouter::new()
+//!     // Register consumers via .consumer(handler)
 //! }
 //! ```
 
 use reinhardt::url_patterns;
+use reinhardt::WebSocketRouter;
 
 use crate::config::apps::InstalledApp;
 
 #[url_patterns(InstalledApp::{{ app_name }}, mode = ws)]
-pub fn ws_url_patterns() {
+pub fn ws_url_patterns() -> WebSocketRouter {
+    WebSocketRouter::new()
     // Register WebSocket consumers here.
-    // Example: .consumer("chat/", chat_consumer)
+    // Example: .consumer(chat_consumer)
 }

